src/OrchardCore.Modules/OrchardCore.Widgets/OrchardCore.Widgets.csproj

```diff
@@ -18,6 +18,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Data.Abstractions\OrchardCore.Data.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement\OrchardCore.ResourceManagement.csproj" />
 
```

Comment:
Remove that extra line

Reply:
It seems you need a direct reference to the assembly when in release mode.
4 modules reference this way. (The extra line seems to be a convention in all)

I’d rather move the tag helper to OrchardCore.Contents.Core as you say re circular references but that becomes a breaking change

If ok with the breaking change let me know and I adjust

Reply:
Ok yes I think it would be better to move this to the abstraction project unless @sebastienros doesn't agree. 😅

Reply:
to .Core is fine, but not abstractions

Reply:
Done, involved changing a few references around. But they now reference OrchardCore.Contents.Core` directly, rather than a reference on `OrchardCore.Contents`